.github/workflows/check-learning-path-links.yml

```diff
@@ -0,0 +1,79 @@
+name: 'Check Learning Path Links'
+on:
+  schedule: # Run once a month
+    - cron: '0 0 1 * *'
+  workflow_dispatch:
+
+permissions: {}
+  
+jobs:
+  check-learning-path-links:
+    name: 'Check Learning Path Links'
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+
+    steps:
+      - name: Checkout head
+        uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+          fetch-depth: 0 # Fetch the entire repo for the below git commit graph operations
```

Comment:
This shouldn't be needed, there's no git graph operations that are being performed

Reply:
The comment was inaccurate, but I actually added this intentionally in my testing because there was an instance of my prior commit not being found if the fetch depth wasn't set.